spec/models/healthcheck_spec.rb

```diff
@@ -92,6 +92,12 @@
   describe '.check' do
     subject { described_class.check }
 
-    it { is_expected.to include sidekiq: false }
+    before do
```

Comment:
I don't get it but it's fine

Reply:
Just verifies that it cascades through from `Healthcheck.check` to `Healthcheck.instance.check`

Reply:
The fix is because depending upon ordering, the redis key may already be set from an earlier spec - which then results in a `true` instead of a `false` - explicitly setting the key to a known value before hand solves that